src/i18n/fr_FR.ts

```diff
@@ -70,736 +70,442 @@ export default {
-            color: '#0016a3',
-            coins: 15,
-            credits: 20_000,
-            minPersonnel: 0,
-            maxPersonnel: 0,
-            schooling: 'Fire Station - Heavy Machinery Operating',
-            shownSchooling: 'Heavy Machinery Operating',
-            special:
-                "Dozer Trailer that can be towed with the Crew cap semi. Requires Heavy Machinery Operating and Truck Driver's License trained personnel to operate.<br> The dozer trailer cannot be assigned personnel, <br>either assign your trained personnel to the towing vehicle, or assign them to a separate unit en route.",
-        },
-        {
-            caption: 'Crew cap semi',
-            color: '#0018b8',
-            coins: 12,
-            credits: 5_000,
-            minPersonnel: 1,
-            maxPersonnel: 3,
-            schooling: "Fire Station - Truck Driver's License",
-            shownSchooling: "Truck Driver's License",
-        },
-        {
-            caption: 'FBI Unit',
-            color: '#001bcc',
-            coins: 15,
-            credits: 10_000,
-            minPersonnel: 2,
-            maxPersonnel: 4,
-        },
-        {
-            caption: 'FBI Investigation Wagon',
-            color: '#001ee0',
-            coins: 15,
-            credits: 10_000,
-            minPersonnel: 2,
-            maxPersonnel: 4,
-        },
-        {
-            caption: 'FBI Mobile Command Center',
-            color: '#0021f5',
-            coins: 25,
-            credits: 25_000,
-            minPersonnel: 1,
-            maxPersonnel: 4,
-            schooling: 'Police - FBI Mobile Center Commmander',
-            shownSchooling: 'FBI Mobile Center Commmander',
-        },
-        {
-            caption: 'FBI Bomb Technician Vehicle',
-            color: '#0a2bff',
-            coins: 35,
-            credits: 35_500,
-            minPersonnel: 1,
-            maxPersonnel: 2,
-            schooling: 'Police - FBI Bomb Technician',
-            shownSchooling: 'FBI Bomb Technician',
-        },
-        {
-            caption: 'FBI Surveillance Drone',
-            color: '#f04242',
-            coins: 25,
-            credits: 25_000,
+            caption: 'Unité motocycliste',
+            color: '#aa2255',
+            coins: 18,
+            credits: 2_500,
             minPersonnel: 1,
             maxPersonnel: 1,
             wtank: 0,
-            schooling: 'Police - FBI Drone Operator',
-            shownSchooling: 'FBI Drone Operator',
-        },
-        {
-            caption: 'Sheriff Unit',
-            color: '#ad0e0e',
-            coins: 20,
-            credits: 15_000,
-            minPersonnel: 1,
-            maxPersonnel: 2,
-            schooling: 'Police - Sheriff',
-            shownSchooling: 'Sheriff',
+            schooling: 'Formation motocycliste',
+            shownSchooling: 'Policie motocycliste',
         },
+                
     ],
     buildings: [

```

Comment:
Same as vehicles
buildings: **{**
id: {

And change the credits to a number without the _ so 100_00 will be 100000, the same at vehicles